tests/e2e/configs/inversify.types.ts

```diff
@@ -48,27 +48,24 @@ const CLASSES = {
     CheGitApi: 'CheGitApi',
     GitPlugin: 'GitPlugin',
     NotificationCenter: 'NotificationCenter',
-    PreferencesHandlerTheia: 'PreferencesHandlerTheia',
     CheApiRequestHandler: 'CheApiRequestHandler',
     CreateWorkspace: 'CreateWorkspace',
     OpenDialogWidget: 'OpenDialogWidget',
     UpdateAccountInformationPage: 'UpdateAccountInformationPage',
     KubernetesPlugin: 'KubernetesPlugin',
     BrowserTabsUtil: 'BrowserTabsUtil',
     PluginsView: 'PluginsView',
-    LanguageServerTestsTheia: 'LanguageServerTestsTheia',
-    CodeExecutionTestsTheia: 'CodeExecutionTestsTheia',
-    ProjectAndFileTestsTheia: 'ProjectAndFileTestsTheia',
-    ProjectAndFileTestsCheCode: 'ProjectAndFileTestsCheCode',
-    WorkspaceHandlingTests: 'WorkspaceHandlingTests',
+    ProjectAndFileTests: 'ProjectAndFileTests',
```

Comment:
I would still maintain the CheCode naming convention, we may need to add other editors later on c.c.: @dmytro-ndp

Reply:
@ScrewTSW : it was actual before previous week when it Mario and Ilya had asked to keep as minimum GUI tests as possible.
So, it doesn't look like we will support something more then smoke Che Code Editor E2E GUI test in near future.

Reply:
Now in Che we have only one userstory for Che-Code (and in general too) - EmptyWorkspace.spec.ts. And only one Class was marked by CheCode - this one

Reply:
ACK, THX